libs/base/ui/action/icon-button/src/icon-button.styles.ts

```diff
@@ -6,7 +6,7 @@ export const iconButtonStyles = css`
     align-items: center;
     gap: 8px;
     cursor: pointer;
-    color: var(--oryx-color-neutral-11);
+    color: var(--oryx-button-color, var(--oryx-color-neutral-11));
```

Comment:
You can control color of the icon by `--oryx-icon-color` css var. Just apply in the styles